bciers/libs/shared/components/src/lib/datagrid/DataGrid.tsx

```diff
@@ -51,13 +51,15 @@ const DataGrid: React.FC<Props> = ({
   const { replace } = useRouter();
   const isRowsEmpty = rows.length === 0;
 
+  // brianna I don't think we need this, the second use effect should do it
```

Comment:
It's been awhile since I developed this but I'm not sure if the `searchParams` dependency array ran every time or just when there was search params. I think it may work just adding `setIsComponentMounted(true)` in the second useEffect, though there is the possibility that will cause just pass the ` if (!isComponentMounted || !fetchPageData) return;` the first time around.

Having them separate ensures that won't happen and is just clearer and more intentional.